syntax = "proto2";

package zk;

message RegisterInfo {
  optional string ip = 1;
  optional int32 port = 2;
}

message SubscriberInfo {
  optional string service_prefix = 1;
}

message ClusterInfo {
  optional string url = 1;
  optional int32 timeout = 2;
  optional string service_prefix = 3;
}
